src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -224,11 +219,12 @@ class ReportingConfigForm extends React.Component {
                 disabled={config && !dataTypesOptionsValues.includes(config.dataType)}
                 options={[...dataTypesOptions, ...selectedDataTypesOption]}
```

Comment:
After migrating this deprecated `Input` to `Form.Control`, these `options` will need to be passed via `<option>` HTML tags rendered as children to `Form.Control`.